### PR TITLE
feat(@angular/cli): bump blueprint dependencies

### PR DESCRIPTION
**Pull request by danielsogl**

### CODE REVIEW

**Comment by hansl** [comment]
@filipesilva  This is a feature and will have to wait for 1.3.

**Comment by danielsogl** [comment]
@hansl thank you for pointing me out! I changed the PR title.

**Comment by clydin** [comment]
Note also that the current versioning will install the latest versions (within the current major).

File: packages/@angular/cli/blueprints/ng/files/package.json [hunk 1]
```diff
@@ -15,26 +15,26 @@
...
   "devDependencies": {
     "@angular/cli": "<%= version %>",
-    "@angular/compiler-cli": "^4.0.0",
-    "@angular/language-service": "^4.0.0",<% if (!minimal) { %>
-    "@types/jasmine": "2.5.45",
-    "@types/node": "~6.0.60",
-    "codelyzer": "~3.0.1",
+    "@angular/compiler-cli": "^4.2.4",
+    "@angular/language-service": "^4.2.4",<% if (!minimal) { %>
+    "@types/jasmine": "2.5.53",
```
Comment (clydin) [inline packages/@angular/cli/blueprints/ng/files/package.json]: The jasmine types can't be updated in isolation.  See PR #6799 for details.

File: packages/@angular/cli/blueprints/ng/files/package.json [hunk 2]
```diff
@@ -15,26 +15,26 @@
...
+    "@types/node": "~8.0.3",
```
Comment (clydin) [inline packages/@angular/cli/blueprints/ng/files/package.json]: This should track node v6 to support the minimum version required by the CLI.

File: packages/@angular/cli/blueprints/ng/files/package.json [hunk 3]
```diff
@@ -15,26 +15,26 @@
...
+    "zone.js": "^0.8.12"
   },
   "devDependencies": {
     "@angular/cli": "<%= version %>",
-    "@angular/compiler-cli": "^4.0.0",
-    "@angular/language-service": "^4.0.0",<% if (!minimal) { %>
-    "@types/jasmine": "2.5.45",
+    "@angular/compiler-cli": "^4.2.4",
+    "@angular/language-service": "^4.2.4",<% if (!minimal) { %>
+    "@types/jasmine": "2.5.53",
```
Comment (cexbrayat) [inline packages/@angular/cli/blueprints/ng/files/package.json]: This will break e2e tests compilation, unless #6799 is merged

**Comment by filipesilva** [comment]
Heya, you'll have to rebase on top of master since https://github.com/angular/angular-cli/pull/6799 was merged just now. You also have to rebase to a single commit since in order to pass the commit message check.

**Comment by danielsogl** [comment]
https://github.com/angular/angular-cli/pull/6817

**Pull request closed without merging**